recipes/leftbar/contact_row/contact_row.vue

```diff
@@ -247,3 +247,7 @@ export default {
   },
 };
 </script>
+
+<style lang="less" scoped>
+@import "../style/leftbar_row.less";
+</style>
```

Comment:
This is not necessary as this component is using `dt-recipe-general-row` as base, which already includes the stylings

Reply:
It seems to actually need this. Scoped isolates the styles to only the general_row component, so any of the classes in leftbar_row.less can not be used in contact_row